modules/@apostrophecms/widget-type/ui/apos/components/AposWidget.vue

```diff
@@ -15,3 +18,15 @@ export default {
 };
 
 </script>
+
+<style scoped>
+/* TODO: change default placeholder style */
+.apos-placeholder {
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(149, 0, 0, 0.579);
+}
```

Comment:
TODO: change default placeholder style, or remove it any style attached to the `apos-placeholder` class? (no default style)